infrastructure/charts/db-exporting/values.yaml

```diff
@@ -6,7 +6,7 @@ image:
   repository: registry.datexis.com/greendb/db-exporting
   pullPolicy: Always
   # Overrides the image tag whose default is the chart appVersion.
-  tag:
+  tag: test
```

Comment:
why do we default to test here?

Reply:
I set it because that's how we have it also in the start_job values.yaml, for which we do not upload the image to github container registry either.